bigquery.js

```diff
@@ -0,0 +1,93 @@
+var BigQueryClient = require('google-cloud/node_modules/@google-cloud/bigquery');
+
+const projectData = require('./auth.json');
+
+const config = {
+  projectId: projectData['project_id'],
+  keyFilename: './auth.json'
+};
+
+const ds = 'atomic';
+
+const PARTITION_FIELDS = ['updated_at', 'created_at', 'fired_at', 'ts'];
```

Comment:
It is not better use this inside a env variable? Doing this, your code will be more reusable and generic 👍 

`const PARTITION_FIELDS = ENV['PARTITION_FIELDS'].split(',')`

Reply:
@renatomenegasso https://github.com/Bidu/snowplow-bigquery-lambda-loader/pull/3/commits/ebb3b00dffef48f1fcd72ed160432bfd2f277db5